client/src/components/ProductOverview/AddToCart.jsx

```diff
@@ -5,7 +5,23 @@ const AddToCart = () => {
 
   return (
     <div id={s.cart}>
-      <h1>AddToCart</h1>
+      <select id={s.size}>
+        <option>SELECT SIZE</option>
```

Comment:
Make sure as we talked in the zoom, once you've done with handling style selector stuff, change hardcoding please 👍